src/services/etherpad/hooks/Pad.js

```diff
@@ -0,0 +1,59 @@
+const { authenticate } = require('@feathersjs/authentication').hooks;
+const { disallow } = require('feathers-hooks-common');
+const { Forbidden } = require('@feathersjs/errors');
+
+const logger = require('../../../logger');
+
+const { hasPermission } = require('../../../hooks');
+const globalHooks = require('../../../hooks');
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected 1 empty line after require statement not followed by another require.](https://app.codacy.com/manual/schul-cloud/schulcloud-server/pullRequest?prid=5525245)